source/reference/minio-mc-admin/mc-admin-idp-ldap-policy.rst

```diff
@@ -0,0 +1,87 @@
+.. _minio-mc-admin-idp-ldap-policy:
+
+============================
+``mc admin idp ldap policy``
+============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin idp ldap policy
+
+Description
+-----------
+
+.. start-mc-admin-idp-ldap-policy-desc
+
+The :mc-cmd:`mc admin idp ldap policy` command allows you to view the mapping relationships between policies and the associated groups or users.
```

Comment:
Am I wrong that the LDAP/AD user has to have authenticated *at least once* to MinIO for this to work? I vaguely recall that being a thing.

Reply:
Checking with others.

Reply:
Per @vadmeste, this is not needed. In fact, the mapping has to be in place before the user can log in.